File: frontend/ProjectManagement.Core/Models/CreateSprintRequest.cs
namespace ProjectManagement.Core.Models;                                                                                                         
                                                                                                                                                   
public sealed record CreateSprintRequest                                                                                                         
{                                                                                                                                                
    public required Guid ProjectId { get; init; }                                                                                                
    public required string Name { get; init; }                                                                                                   
    public string? Goal { get; init; }                                                                                                           
    public required DateTime StartDate { get; init; }                                                                                            
    public required DateTime EndDate { get; init; }                                                                                              
}

File: backend/pm-server/src/api/sprints/create_sprint_request.rs
use serde::Deserialize;

/// Request body for creating a sprint
#[derive(Debug, Deserialize)]
pub struct CreateSprintRequest {
    pub project_id: String,
    pub name: String,

    #[serde(default)]
    pub goal: Option<String>,

    /// Unix timestamp (seconds since epoch)
    pub start_date: i64,

    /// Unix timestamp (seconds since epoch)
    pub end_date: i64,
}
